sailthru_content/services/tests/test_sailthru_api_service.py

```diff
@@ -0,0 +1,133 @@
+#! /usr/bin/env python
+import json
+import mock
+import unittest
+
+from factory import fuzzy
```

Comment:
FactoryBoy now includes the `factory.Faker` class. Built-in fuzzers are deprecated in favor of their [Faker](https://factoryboy.readthedocs.io/en/latest/#realistic-random-values) equivalents.